pkg/services/networkverifier/networkverifier.go

```diff
@@ -97,8 +98,8 @@ type VerifierResult int
 // Verifier outcomes
 const (
 	Undefined VerifierResult = 0
-	Failure
-	Success
+	Failure   VerifierResult = 1
+	Success   VerifierResult = 0
```

Comment:
**question:**
How does this even compile? :surprisedpikachu:  we have 2 `VerifierResults` with value 0